frappe/public/js/frappe/ui/field_group.js

```diff
@@ -125,11 +125,14 @@ frappe.ui.FieldGroup = frappe.ui.form.Layout.extend({
 		return this.set_value(key, val);
 	},
 	set_values: function(dict) {
+		let promises = [];
```

Comment:
@surajshetty3416 @netchampfaris I'll need your review on this change.
`set_value` returns a promise, logic would dictate `set_values` to return a promise as well